streams/src/test/java/org/apache/kafka/streams/processor/internals/StreamsProducerTest.java

```diff
@@ -144,21 +152,81 @@ public void shouldFailIfProducerIsNull() {
         {
             final NullPointerException thrown = assertThrows(
                 NullPointerException.class,
-                () -> new StreamsProducer(null, true, logContext)
+                () -> new StreamsProducer(null, EXACTLY_ONCE_ALPHA, logContext)
+            );
+
+            assertThat(thrown.getMessage(), is("producer cannot be null"));
+        }
+
+        {
+            final NullPointerException thrown = assertThrows(
+                NullPointerException.class,
+                () -> new StreamsProducer(null, EXACTLY_ONCE_BETA, logContext)
             );
 
             assertThat(thrown.getMessage(), is("producer cannot be null"));
         }
     }
 
     @Test
-    public void shouldFailIfLogContextIsNull() {
+    public void shouldFailIfProcessingModeIsNull() {
```

Comment:
This is just a bad diff (the test is not removed, but just the next after this new one)